quizApp.js

```diff
@@ -159,34 +160,71 @@ function displayQuestion(item, append = false) {
   if (!append) {
     section.innerHTML = "";
   }
-  const question = document.createElement("div");
-  section.appendChild(question);
-  const p = document.createElement("p");
-  p.innerText = item.question;
-  question.appendChild(p);
-  item.options.forEach((option) => {
-    const optionBtn = document.createElement("button");
-    optionBtn.classList.add("btnList");
-    optionBtn.innerText = option.text;
-    optionBtn.dataset.correct = option.isCorrect;
-    question.appendChild(optionBtn);
-    optionBtn.addEventListener("click", (event) =>
-      checkAnswer(event, question)
-    );
+  item.forEach((ques) => {
+    const question = document.createElement("div");
+    section.appendChild(question);
+
+    const p = document.createElement("p");
+    p.className = "ques";
+    p.innerText = ques.question;
+    question.appendChild(p);
+
+    ques.options.forEach((option) => {
+      const optionBtn = document.createElement("button");
+      optionBtn.classList.add("btnList");
+      optionBtn.innerText = option.text;
+      optionBtn.dataset.correct = option.isCorrect;
+      question.appendChild(optionBtn);
+      optionBtn.addEventListener("click", (event) =>
+        checkAnswer(event, question)
+      );
+    });
+    const solutionBtn = document.createElement("button");
+    question.appendChild(solutionBtn);
+    solutionBtn.classList.add("solutionbtn");
```

Comment:
Pay attention to casing in class names :)

Reply:
I have changed it to solution-button, I have used prettier for formatting and left some empty lines between the code to make it easier to understand.